backend/src/main/java/WebApi.java

```diff
@@ -1,35 +1,57 @@
+import api.customer.CustomerController;
+import api.MaterialInterface;
+import api.PrintingInterface;
+import api.customer.CustomerAPI;
 import com.google.gson.Gson;
 import model.Customer;
+import model.Order;
+import model.OrderedPart;
 
 import static spark.Spark.*;
 
 /**
  * Starts a restapi att localhost:4567
  */
 public class WebApi {
-    private static WebInterface webInterface = new WebInterfaceImpl();
+    private static CustomerAPI ci = new CustomerController();
+    private static MaterialInterface mi;
+    private static PrintingInterface pi;
+
+    private static Gson gson = new Gson();
 
     public static void main(String[] args) {
+        long start = System.currentTimeMillis();
+        System.out.println("STARTED ENDPIONT SETUP");
         WebApi.enableCORS("*","*","*");
 
-        Gson gson = new Gson();
         get("/hello", (req, res) -> "Hello World");
 
-        // Customers
-        get("/customers", (request, response) -> webInterface.getAllCustomers(), gson::toJson);
-        get("/customers/:customerID", ((request, response) -> webInterface.getCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/orders,", ((request, response) -> webInterface.getOrdersFromCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/digitalparts,", ((request, response) -> webInterface.getDigitalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
-        get("/customers/:customerID/physicalparts,", ((request, response) -> webInterface.getPhysicalPartsFromCustomer(request.params(":customerID"))), gson::toJson);
-        post("/customers", ((request, response) -> webInterface.createNewCustomer(gson.fromJson(request.body(), Customer.class))), gson::toJson);
-        put("/customers", ((request, response) -> webInterface.updateCustomer(gson.fromJson(request.body(), Customer.class))), gson::toJson);
-        delete("/customers/:customerID", ((request, response) -> webInterface.deleteCustomer(request.params(":customerID"))), gson::toJson);
-
-        //get("/digitalPart/:digitalPartID", (req, res) -> webInterface.getDigitalPart(req.params("digitalPartID")), gson::toJson);
+        setupCustomerInterface();
 
+        System.out.println("ENDPOINT SETUP COMPLETE: " + (System.currentTimeMillis()-start) + " ms");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)